### PR TITLE
Doc how to add NuGet support to project systems

### PR DESCRIPTION
**Pull request by AArnott**
Please do *not* merge this before NuGet/NuGet.VisualStudioExtension#51 is merged.

### CODE REVIEW

**Comment by yishaigalatzer** [comment]
You want to add that the support is available for CPS projects in NuGet V3, and the support for other project systems was/will be added in nuget 3.2

**Comment by AArnott** [comment]
Done.

File: NuGet.Docs/Create/Project-System-Support.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+[Learn more about project capabilities][1].
+
+Note that for [CPS-based project systems][4], the implementation details
+for project capabilities described in the rest of this section have been done
+for you. Learn more about [declaring project capabilities in CPS projects][5].
+
+### Responding to queries for VSHPROPID_ProjectCapabilitiesChecker
+
+Your project declares this capabilities by supporting the `IVsHierarchy::GetProperty`
```
Comment (GearedToWar) [inline NuGet.Docs/Create/Project-System-Support.md]: 'this'->'these'

Comment (GearedToWar) [inline NuGet.Docs/Create/Project-System-Support.md]: 'this'->'these'

**Comment by AArnott** [comment]
@GearedToWar: fixed by #365

**Pull request merged**